playbooks/upload_to_swift.yml

```diff
@@ -0,0 +1,90 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: False
+  tasks:
+    - name: Create archive
+      command: "tar -cjf {{ archive_name }} {{ artifacts_dir }}"
+      args:
+        chdir: "{{ artifacts_parent_dir }}"
+      tags:
+        - skip_ansible_lint
+
+    - name: Create a public Cloud Files container
+      os_object:
+        container: "{{ container }}"
+        container_access: "public"
+        region_name: "{{ region }}"
+        cloud: "{{ cloud_name }}"
+
+    - name: Authenticate to the cloud and retrieve the service catalog
+      os_auth:
+        cloud: "{{ cloud_name }}"
+        region_name: "{{ region }}"
```

Comment:
Any issues/downsides to having artifacts scattered across multiple regions?  I'm not aware of any (and will presumably make uploading faster since you're uploading from an instance in the same region), but just pointing it out incase anyone can think of any.

Reply:
The region is set to `"DFW"` at the bottom of this file so this change should keep things as they currently are.

Reply:
Ah ok, thought that was coming in from Jenkins.